### PR TITLE
446 view reward panel Frontend

### PR DESCRIPTION
**Pull request by topocount**
The reward panel styling is ready for review. I did a lot more styling here than I've every done on this project before so please let me know if there are any best-practices I should incorporate.

### CODE REVIEW

File: apps/rewards/app/components/Panel/ViewReward.js [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+            <div>
+              <h2>
+                <Label>Type</Label>
+              </h2>
+              <div>
+                {isMerit?'Merit Reward':'Dividend Reward'}
+              </div>
+            </div>
+          </SidePanelSplit>
+          <SidePanelSplit>
```
Comment (rkzel) [inline apps/rewards/app/components/Panel/ViewReward.js]: Needs to be:
`        </SidePanelSplit>
`          <SidePanelSplit style={{ borderTop: '0' }}>

or else there is double-width border.

File: apps/rewards/app/components/Panel/ViewReward.js [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+            <TokenIcon />
+            <Summary>
+              <p>
+                {'A total of '}
+                <SummaryVar>{displayCurrency(amount)} {translateToken(rewardToken)}</SummaryVar>
+                {' will be distributed as a reward to addresses that earned '}
+                <SummaryVar>{referenceToken}</SummaryVar>
+                {' from '}
+                <SummaryVar>
+                  {startDate.getDate().toString() + '-' + MONTHS[startDate.getMonth()] + '-' + startDate.getFullYear()}
```
Comment (rkzel) [inline apps/rewards/app/components/Panel/ViewReward.js]: there is formatting utility in date-fns...

`import { format } from 'date-fns'`
`format(startDate, 'dd-MMM-yyyy')`

File: apps/rewards/app/components/Panel/ViewReward.js [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+        isMerit,
+        referenceToken,
+        rewardToken,
+        amount,
+        startDate,
+        endDate,
+        description,
+        delay,
+      } = reward
+      const truncatedCreator = `${creator.slice(0, 6)}...${creator.slice(-4)}`
```
[the remark this answers is not in the document]
Comment (topocount) [reply apps/rewards/app/components/Panel/ViewReward.js]: Template Literals are fun! (and I borrowed this from Otto)

The creator will indeed always be an address

File: apps/rewards/app/components/Panel/ViewReward.js [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+      const translateToken = (token) => {
+        if (token == 0x0) {
+          return 'ETH'
+        }
```
[the remark this answers is not in the document]
Comment (topocount) [reply apps/rewards/app/components/Panel/ViewReward.js]: It needs to be manual for ETH. For other _actual_ tokens we'll be able to query the symbol from the token contract in script.js

File: apps/rewards/app/components/Panel/ViewReward.js [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+
+
+
+class ViewReward extends React.Component {
+    static propTypes = {
+      reward: PropTypes.object,
+    }
+    renderDescription = (description = '') => {
+      // Make '\n's real breaks
+      return description.split('\n').map((line, i) => (
```
[the remark this answers is not in the document]
Comment (topocount) [reply apps/rewards/app/components/Panel/ViewReward.js]: We could in theory just replace `\n` with a `<br>` tag but this is how render descriptions elsewhere

File: apps/rewards/app/components/Panel/ViewReward.js (hunk 1 above)
Comment (topocount) [reply apps/rewards/app/components/Panel/ViewReward.js]: Thanks! I was wondering about the best way around this

**Pull request merged**